mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -45,9 +47,26 @@ class SetpointRawPlugin : public plugin::PluginBase,
 
 		bool tf_listen;
 
+		ignore_rpyt_messages_ = false;
```

Comment:
init this on the constructor instead: https://github.com/mavlink/mavros/pull/964/files#diff-146253845b95f1e30b7d5bd15e269768R41